app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -322,7 +343,7 @@ class PipelineSampleReads extends React.Component {
                 .slideUp(200);
             } else {
               $(".note-save-failed")
-                .html(`<i class='fa fa-frown-o'></i> ${response.data.message}`)
+                .html(`<i class='fa fa-frown-o'></i> ${response.data.message} ${response.data.errors}`)
```

Comment:
This comment is outside the scope of this PR, but in general we should not use jQuery($) to update the DOM, because it defeats the purpose of using React in the first place (which is to enjoy the virtual DOM diff and only update the minimal amount of DOM).

And this also messes up with React's render cycle, since React is unaware of the DOM changes made by jQuery($), so next time React re-renders this component in virtual DOM, even if this part of the DOM node has not changed, jQuery will still touch the DOM unnecessarily)

Just headsup :)